course/analytics.py

```diff
@@ -97,14 +99,29 @@ def add_data_point(self, value, weight=1):
             self.string_weights[value] = \
                     self.string_weights.get(value, 0) + weight
         elif value is None:
-            self.add_data_point("(None)", weight)
+            self.add_data_point(
+                "".join([
+                        "(",
+                        pgettext("No data","None"),
+                        ")"]),
+                weight)
         else:
             if (self.num_max_value is not None
                     and value > self.num_max_value):
-                self.add_data_point("(value greater than max)", weight)
+                self.add_data_point(
+                    "".join([
+                            "(",
+                            pgettext("Value of grade", "value greater than max"),
+                            ")"]),
+                    weight)
```

Comment:
Just don't want distract the translator from characters which don't require translation. Sometimes I forget to put parentheses in translated text, or spaces at the end of a string, and then it might result in a bad looking.